### PR TITLE
fix: set pending approval status when the same is set in UserSignup

### PR DESCRIPTION
**Pull request by MatousJobanek**
set pending approval status when the same is set in UserSignup `approved` condition type - no matter what is set in `complete` condition

### CODE REVIEW

File: pkg/signup/service/signup_service.go [hunk 1]
```diff
@@ -183,8 +183,9 @@ func (s *ServiceImpl) GetSignup(userID string) (*signup.Signup, error) {
 	}
 
 	// Check UserSignup status to determine whether user signup is complete
-	signupCondition, found := condition.FindConditionByType(userSignup.Status.Conditions, v1alpha1.UserSignupComplete)
-	if !found {
+	approvedCondition, approvedFound := condition.FindConditionByType(userSignup.Status.Conditions, v1alpha1.UserSignupApproved)
+	signupCondition, completeFound := condition.FindConditionByType(userSignup.Status.Conditions, v1alpha1.UserSignupComplete)
```
Comment (xcoulon) [inline pkg/signup/service/signup_service.go:187]: to be consistent with the `approvedCondition` variable above, shouldn't we rename `signupCondition` to `completeCondition`?

Comment (MatousJobanek) [reply pkg/signup/service/signup_service.go]: good point, done: 37d3f15

**Review by alexeykazakov — APPROVED** [review]

**Review by xcoulon — APPROVED** [review]

**Pull request merged**